packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -54,6 +55,10 @@ export type ChatCompositeOptions = {
    * @defaultValue true
    */
   topic?: boolean;
+  /**
+   * flag to force focus on the chat send box text field
```

Comment:
nit: update this comment slightly. We only set focus on mount - and could be any of the possible options not just the send box (currently its either sendbox or none but we can be agnostic of that in the comment).

```suggestion
   * Set focus on the composite when the composite first mounts.
```